uri-scheme.md

```diff
@@ -15,26 +21,23 @@ Applications MUST parse all query fields specified below and fail if they contai
 * `version`: The version of the URI scheme.  At the moment, only version 0 is specified. Clients must check this for compatibility.
 Default: `0`
 
-* `appid`: The application ID. This must be passed to the relay server.
-Default: `lothar.com/wormhole/text-or-file-xfer`
-
 * `rendezvous`: The rendezvous server and protocol to use, including its port. The versioned endpoint is supposed to be added by the client implementation, eg. `/v1`
 Default: `ws://relay.magic-wormhole.io:4000`
 
-* `type`: The type of operation requested. Valid values are:
-		* `recv`: The URI generating client is asking to send a file and is requesting the URI parsing client to receive a file.
-		* `send`: The URI generating client is asking to receive a file and is requesting the URI parsing client to offer to send a file.
-Default: `recv`
+* `role`: The type of operation requested. Valid values are:
+		* `follower`: The URI *generating* client is asking to lead the transit connection (and most likely send a file) and is requesting the URI *parsing* client to follow the leader (and most likely receive a file).
+		* `leader`: The URI *parsing* client is supposed to lead the transit connection (and most likely send a file) and the URI *generating* client is asking to follow the leader (and most likely receive a file). This functionality can be used if it is easier (from the user's point of view) to read the URI in the opposite direction (for example because QR codes are used and only one device is equipped with a camera). Clients with URI support are required to support this functionality on the parsing side, but must not necessarily allow to generate such a URI itself.
+Default: `follower`
 
 ## Examples
 To encode the code `4-hurricane-equipment` to send a file the URI would look like this:
 
-`wormhole:4-hurricane-equipment`
+`wormhole-transfer:4-hurricane-equipment`
 
 By expanding the default values this would be equivalent to:
 
-`wormhole:4-hurricane-equipment?version=0&appid=lothar.com%2Fwormhole%2Ftext-or-file-xfer&rendezvous=ws%3A%2F%2Frelay.magic-wormhole.io%3A4000%2Fv1&type=recv`
+`wormhole-transfer:4-hurricane-equipment?version=0&appid=lothar.com%2Fwormhole%2Ftext-or-file-xfer&rendezvous=ws%3A%2F%2Frelay.magic-wormhole.io%3A4000%2Fv1&role=leader`
```

Comment:
+1, like "role" better (and matches other langauge in magic-wormhole)

Reply:
I forgot to remove appid from this URI though